code/game/turfs/space/transit.dm

```diff
@@ -91,3 +91,6 @@
 
 	icon_state = "speedspace_ns_[state]"
 	transform = turn(matrix(), angle)
+
+/turf/open/space/rcd_vals(mob/user, obj/item/weapon/rcd/the_rcd)
```

Comment:
This looks like you're applying it to all space turfs, rather than transit space turfs.